CHANGELOG.md

```diff
@@ -69,6 +69,20 @@ Use the classes starting with `.govuk-!-static-margin` and `.govuk-!-static-padd
 
 This change was introduced in [pull request #3593: Remove deprecated static spacing classes](https://github.com/alphagov/govuk-frontend/pull/3593).
 
+#### Remove the fallback GOV.UK crown logo from your HTML
+
+The header component previously included a fallback version of the GOV.UK crown logo for Internet Explorer 8. As Frontend no longer supports IE8, this fallback has been removed.
+
+If you're not using the Nunjucks macros, you'll need to remove this fallback from your HTML code. In your header component:
+
+1. Remove the block of HTML containing the `govuk-header__logotype-crown-fallback-image` image (starting with `<!--[if IE 8]>` and ending with `<![endif]-->`).
+2. Remove `<!--[if gt IE 8]><!-->` and ` <!--<![endif]-->` from around the `govuk-header__logotype-crown` SVG, but don't remove the SVG.
+3. Delete the `govuk-logotype-crown.png` file from your assets folder.
+
+You don't need to change any HTML if you're using the supplied Nunjucks macros, but will still need to remove the `govuk-logotype-crown.png` image.
```

Comment:
Depending on how a service is set up, they might be e.g. copying the files from our assets folder or proxying requests directly to our package, so teams _might_ not have to do anything to remove the PNG.

Reply:
Do you think it's a case of changing this to "you may still need to remove" or do we want something more detailed?

Reply:
I think that'd probably be enough 👍🏻 Could maybe link to [the related docs](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#font-and-image-assets)? Maybe something like…

> You don't need to change any HTML if you're using the supplied Nunjucks macros, but may still need to remove the `govuk-logotype-crown.png` image depending on [how you are serving the font and image assets](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#font-and-image-assets).